src/class/ServerManager/ServerManager.cpp

```diff
@@ -204,10 +220,10 @@ ServerManager::status ServerManager::setup()
 			(*it)->bindSockets();
 			(*it)->listenSockets();
```

Comment:
if listen fails, do we need to do something about the binded sockets?

Reply:
As far as I  know, closing the fd used to bind is enough to "unbind" it